subprojects/tooling-api/tooling-api.gradle.kts

```diff
@@ -0,0 +1,121 @@
+import accessors.*
+import org.gradle.gradlebuild.BuildEnvironment
+import org.gradle.gradlebuild.buildquality.testfiles.TestFileCleanUpExtension
+import org.gradle.gradlebuild.packaging.ShadedJar
+import org.gradle.gradlebuild.test.integrationtests.IntegrationTest
+import org.gradle.gradlebuild.unittestandcompile.ModuleType
+import org.gradle.plugins.ide.eclipse.model.EclipseClasspath
```

Comment:
:x: unused import

Reply:
Fixed.

Why don't I see the unused imports in IDEA?